### ISSUE TITLE
Add GRAPE

### ISSUE THREAD

**Issue by darabos** [lynxkite/lynxkite#104]
https://github.com/maxiaoba/GRAPE is a state of the art feature imputation / label prediction method. It would be a good fit for LynxKite because it's an excuse to use a graph neural network in a setting where you don't even really have a graph. Also it uses PyTorch Geometric which makes the integration hopefully not too hard.
